### PR TITLE
Added additional logging

### PR DESCRIPTION
**Pull request by rvema**
- Added more log statements
- Set the execution time in seconds
- Added support for upsertTime which can help with automatic data cleanup using mongo ttl-index

### CODE REVIEW

File: src/main/java/com/capitalone/dashboard/collector/DefaultGitHubClient.java [hunk 1]
```diff
@@ -377,7 +377,7 @@ public RedirectedStatus checkForRedirectedRepo(GitHubRepo repo) throws Malformed
         JSONObject queryJSONBody = parseAsObject(response);
         String repoUrl = str(queryJSONBody, "html_url");
         if (!repoUrl.equals(repo.getRepoUrl())) {
-            LOG.info("original url: " + repo.getRepoUrl() + " is redirected to new url: " + repoUrl);
+            LOG.info("original_url=" + repo.getRepoUrl() + ", redirected_url: " + repoUrl);
```
Comment (danielyhuang) [inline src/main/java/com/capitalone/dashboard/collector/DefaultGitHubClient.java:380]: do you want to change the : to = for easy Splunk search?

File: src/main/java/com/capitalone/dashboard/collector/GitHubCollectorTask.java [hunk 2]
```diff
@@ -128,13 +130,6 @@ public GitHubCollector getCollector() {
         uniqueOptions.put(GitHubRepo.BRANCH, "");
         protoType.setUniqueFields(uniqueOptions);
 
-        if (existingCollector != null) {
-            protoType.setLastPrivateRepoCollectionTimestamp(existingCollector.getLastPrivateRepoCollectionTimestamp());
-            protoType.setLastCleanUpTimestamp(existingCollector.getLastCleanUpTimestamp());
```
Comment (danielyhuang) [inline src/main/java/com/capitalone/dashboard/collector/GitHubCollectorTask.java:133]: Do we no need these fields in protoType?

Comment (danielyhuang) [inline src/main/java/com/capitalone/dashboard/collector/GitHubCollectorTask.java:133]: Do we not need these fields in protoType?

File: src/main/java/com/capitalone/dashboard/collector/GitHubCollectorTask.java [hunk 3]
```diff
@@ -128,13 +130,6 @@ public GitHubCollector getCollector() {
...
-            protoType.setLastExecutionRecordCount(existingCollector.getLastExecutionRecordCount());
-            protoType.setLatestProcessedEventId((existingCollector).getLatestProcessedEventId());
-            protoType.setLatestProcessedEventTimestamp((existingCollector).getLatestProcessedEventTimestamp());
-        }
```
Comment (danielyhuang) [inline src/main/java/com/capitalone/dashboard/collector/GitHubCollectorTask.java:137]: Do we not need these fields in protoType?

Comment (rvema) [reply src/main/java/com/capitalone/dashboard/collector/GitHubCollectorTask.java:137]: no need as we are returning the collector as soon as we find it . please check few lines above

File: src/main/java/com/capitalone/dashboard/collector/DefaultGitHubClient.java (hunk 1 above)
Comment (rvema) [reply src/main/java/com/capitalone/dashboard/collector/DefaultGitHubClient.java:380]: sure

**Review by danielyhuang — APPROVED** [review]
lgtm

**Pull request merged**